Documentation_displayed_on_the_portal/Share_Dataset_Instruction_Page.md

```diff
@@ -0,0 +1,222 @@
+# Dataset Addition Procedures
+
+First of all, thank you for sharing your data with CONP and the scientific community!
+
+All datasets to upload to the CONP portal must include the following files in the root directory:
+
+- a required `README.md` file. The content of this file will be displayed in the portal page describing your dataset.
+- a required `DATS.json` file as described in [the main documentation page](https://github.com/CONP-PCNO/conp-documentation/blob/master/CONP_DATS_fields.md). We provide a [DATS GUI editor]() ***(#TODO: ADD LINK TO EDITOR)*** for easy creation of this file. Note: the content of this file will also be displayed on the portal page describing your dataset.
```

Comment:
Were you intending to add the editor link before submitting the PR?

Reply:
Indeed. Right now, there is no link to the editor so I need to figure out how to have a link that directs the user directly to the editor. I'll update that TODO once there is a proper link.